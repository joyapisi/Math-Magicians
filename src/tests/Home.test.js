import React from 'react';
import { render } from '@testing-library/react';
import Home from '../components/Home';

describe('Home', () => {
  test('should render correctly', () => {
    const { container } = render(<Home />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
