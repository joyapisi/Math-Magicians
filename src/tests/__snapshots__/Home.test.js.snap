// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should render correctly 1`] = `
<div
  id="home"
>
  <h3>
    Welcome to our page!
  </h3>
  <p>
    Calculators have not only greatly enhanced our ability to perform the regular computations that are involved in everyday life, but provided humans with the ability to understand mathematics on a greater scale than ever imagined.
  </p>
  <p>
    This technology allows students solve complicated problems quickly and in an efficient manner. Additionally, it can reduce the problem to simpler tasks and allows the student to devote more time in understanding the problem. Secondly, they are saved from monotonous calculations and the same boring mundane procedure.
  </p>
</div>
`;
